src/Ui.GraphiQL/Internal/graphiql.cshtml

```diff
@@ -98,23 +98,22 @@
 			updateURL();
```

Comment:
This updateUrl is used here but you removed the function.

Reply:
got removed mistakenly, added back